README.md

```diff
@@ -1,3 +1,2 @@
-# Visual Studio Code CAPL extension
-
-This repository is for public resources used in the Visual Studio Code CAPL extension.
\ No newline at end of file
+# Visual Studio Code extension - CAPL
+This repository is a placeholder for public resources used in the `CAPL` Visual Studio Code extension. The extension itself is not open source. This repository is only inteded for storing public resources referenced in the marketplace page of the extension. 
```

Comment:
```suggestion
This repository is a placeholder for public resources used in the `CAPL` Visual Studio Code extension. The extension itself is not open source. This repository is only intended for storing public resources referenced in the marketplace page of the extension. 
```

I would start a new line per sentence to improve diffs. The text should still be displayed the same way.